apartment.rb

```diff
@@ -1,2 +1,26 @@
 class Apartment
+	attr_accessor :number, :rooms, :monthly_rent
+
+	def initialize(args={})
+
+		@number = args[:number]
+		@rooms = args[:rooms]
+		@monthly_rent = args[:monthly_rent] || 1000
+	end
+
+	def total_sqft
+		@rooms.map {|room| room.sqft}.reduce(:+)
```

Comment:
Another way to solve this is just by using reduce, without calling map first, which reduces the amount of method calls on @rooms.
```
@rooms.reduce(0) { |sum, room| sum += room.sqft }
```
The 0 is the default starting value and the block says to go through each room and add it's sqft to this sum. The return value is the sum.